src/calculateRentalCost.js

```diff
@@ -28,7 +28,26 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_PER_DAY = 40;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM_DISCOUNT = 50;
+  const LONG_TERM = 7;
+  const SHORT_TERM = 3;
+  let res = 0;
+
+  if (days >= LONG_TERM) {
+    res = RENT_PER_DAY * days - LONG_TERM_DISCOUNT;
+  }
+
+  if (days >= SHORT_TERM && days < LONG_TERM) {
+    res = RENT_PER_DAY * days - SHORT_TERM_DISCOUNT;
+  }
+
+  if (days < SHORT_TERM) {
+    res = RENT_PER_DAY * days;
+  }
```

Comment:
You can return instantly when you calculated the value.
![image](https://github.com/mate-academy/js_task-transportation-on-vacation/assets/63243136/610e0837-fc2a-4b1d-b825-3d01e521431f)